src/types/tokens.ts

```diff
@@ -0,0 +1,55 @@
+import { SignableToken } from '../api';
+
+export interface ERC20Token {
+  type: 'ERC20';
+  tokenAddress: string;
+}
+
+export interface ERC721Token {
+  type: 'ERC721';
+  tokenId: string;
+  tokenAddress: string;
+}
+
+export interface ETHToken {
+  type: 'ETH';
+}
+
+export interface ETHAmount extends ETHToken {
+  amount: string;
+}
```

Comment:
Can we add a newline here.

Reply:
Emacs kept deleting end of file line. I had to turn off the commit hooks to get this commit [8d42a84](https://github.com/immutable/imx-core-sdk/pull/149/commits/8d42a8478a0aec8c6b527dff0ccdd66c8840cf91)